src/V3Number.cpp

```diff
@@ -1011,6 +1011,17 @@ uint32_t V3Number::mostSetBitP1() const {
     }
     return 0;
 }
+
+bool V3Number::eqTwoState(const V3Number& rhs) const {
+    // This is supposed to be fast
+#ifdef VL_DEBUG  // Otherwise slow
```

Comment:
Did you measure this as slow?   It is rare to run with debug so we might be missing an assert that e.g. fires on the regressions.

Reply:
Well, it's only constant overhead I guess, so it matters less with a good hash function.

Reply:
Though if I load the 2 m_valueX arrays I might as well compare them and make this work 4state

Reply:
or drop this function and use an improved isCaseEq instead